.github/workflows/build-package.yml

```diff
@@ -0,0 +1,141 @@
+      - ubuntu-latest
+    steps:
+      # 这里作用是什么？ 设置环境变量？
+      # - uses: actions/checkout@v2
+      # - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+      #   with:
+      #     version: '285.0.0'
+      #     project_id: ${{ secrets.GCP_PROJECT_ID }}
+      #     service_account_email: ${{ secrets.GCP_SA_EMAIL }}
+      #     service_account_key: ${{ secrets.GCP_SA_KEY }}
+      #     export_default_credentials: true
+      #- run: gcloud info
+      - name: Set up Go 1.15
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.15.6
+        id: go
+      - name: setup Docker
+        uses: docker-practice/actions-setup-docker@0.0.1
+        with:
+          docker_version: 20.10
+          docker_channel: stable
+      - name: setup Docker Buildx
+        id: buildx
+        uses: crazy-max/ghaction-docker-buildx@v3
+        with:
+          buildx-version: latest
+          qemu-version: latest 
+      - uses: actions/checkout@v2.1.0
+      - uses: jitterbit/get-changed-files@v1
+        id: changed-files
+        with:
+          format: space-delimited
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/checkout@v2
+      - run: |
+          set -x
+          echo "##输出当前目录"
+          ls -al
+          echo "##clone harbor项目"
+          git clone https://github.com/goharbor/harbor.git src/github.com/goharbor/harbor
+          echo "###输出harbor目录"
+          ls -al src/github.com/goharbor/harbor
+          mkdir -p src/github.com/goharbor/harbor/update
+          mv Makefile src/github.com/goharbor/harbor/update/
+          cd src/github.com/goharbor/harbor 
+          echo "###再出输出harbor目录"
+          ls -al
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_PASSWORD }}
+      - name: Build Base Image
+        # if: contains(steps.changed-files.outputs.modified, 'Dockerfile.base') || contains(steps.changed-files.outputs.modified, 'VERSION')
+        run: |
+          set -x
+          cd src/github.com/goharbor/harbor
+          base_image_tag=$(cat ./VERSION)-arm
+          sudo make -f update/Makefile pre_update
+          sudo make build_base_docker -e BASEIMAGETAG=$base_image_tag -e REGISTRYUSER="${{ secrets.DOCKER_HUB_USERNAME }}" -e REGISTRYPASSWORD="${{ secrets.DOCKER_HUB_PASSWORD }}" -e PUSHBASEIMAGE=yes -e BASEIMAGENAMESPACE=docker.io/harborarmdev -e DOCKER_IMAGE_NAME_PREPARE=harborarmdev/prepare -e DOCKERIMAGENAME_PORTAL=harborarmdev/harbor-portal -e DOCKERIMAGENAME_CORE=harborarmdev/harbor-core -e DOCKERIMAGENAME_JOBSERVICE=harborarmdev/harbor-jobservice -e DOCKERIMAGENAME_LOG=harborarmdev/harbor-log -e DOCKERIMAGENAME_DB=harborarmdev/harbor-db -e DOCKERIMAGENAME_CHART_SERVER=harborarmdev/chartmuseum-photon -e DOCKERIMAGENAME_REGCTL=harborarmdev/harbor-registryctl -e DOCKERIMAGENAME_EXPORTER=harborarmdev/harbor-exporter
```

Comment:
the image name format should be goharbor/harbor-core:${tag}-arm, do not use harborarmdev

Reply:
Here this is used for debugging in the fork's sub-repository. After the debugging is successful, these names will be deleted